client/components/DataManagement/DataManagementRow/index.jsx

```diff
@@ -854,18 +862,10 @@ const DataManagementRow = ({
                             currentDate
                         );
 
-                    const daysBetweenDates = checkTimeBetweenDates(
-                        currentDate,
-                        latestVersion.candidatePhaseReachedAt
-                    );
-                    const DAYS_TO_PROVIDE_FEEDBACK = 120;
                     const shouldShowAdvanceButton =
                         isAdmin &&
                         completedRequiredReports(latestVersion) &&
-                        issuesCount === 0 &&
-                        (recommendedTestPlanVersions.length ||
-                            (!recommendedTestPlanVersions.length &&
-                                daysBetweenDates > DAYS_TO_PROVIDE_FEEDBACK));
+                        issuesCount === 0;
```

Comment:
@alflennik has there been any discussion on also allowing the advance to happen even if there is open issues related to the test plan? #800 correctly assumes that the button wasn't available due to the candidate review time constraint. But an additional factor would be currently open issues (also need to consider that an open issue doesn't have be tied to that specific test plan version, with recent updates).

Do we forsee a need to also bypass that?

cc @lolaodelola

Reply:
Interesting point! I tried to follow requested change as close as I could, but I'd be willing to address that too if there's consensus about it being helpful.